src/utils/patch.ts

```diff
@@ -0,0 +1,284 @@
+// This code contains work that is copyrighted to Joachim Wester (2013)
```

Comment:
@clbond I am assuming that only the minimal code that is needed for Augury was forked into this file from the original repo?